### PR TITLE
Spelling: Ctrl+, media type

### PR DESCRIPTION
**Pull request by comradekingu**
Consistent with https://translate.lxqt-project.org/translate/lxqt-desktop/lximage-qt/sv/?checksum=27f72b4b6a7dea5c (Ctrl+M)

### CODE REVIEW

File: pcmanfm/preferences.ui [hunk 1]
```diff
@@ -899,7 +899,7 @@ A value of -1 means that there is no limit for the file size (the default).</str
             <item>
              <widget class="QCheckBox" name="templateTypeOnce">
               <property name="text">
-               <string>Show only one template for each MIME type</string>
+               <string>Show only one template for each media type</string>
```
Comment (tsujan) [inline pcmanfm/preferences.ui:902]: This isn't about media but mime.

File: pcmanfm/preferences.ui [hunk 2]
```diff
@@ -162,7 +162,7 @@
             <item row="2" column="0" colspan="2">
              <widget class="QCheckBox" name="ctrlRightClick">
               <property name="text">
-               <string>Show folder context menu with Ctrl + right click</string>
+               <string>Show folder context menu with Ctrl+right click</string>
```
Comment (tsujan) [inline pcmanfm/preferences.ui:165]: Unnecessary changes to strings break translations. " + " is as valid as "+".

Comment (comradekingu) [reply pcmanfm/preferences.ui:165]: What it does is fuzzify translations, and alerting translators to how it could be more consistent is in this case warranted.

File: pcmanfm/preferences.ui (hunk 1 above)
Comment (comradekingu) [reply pcmanfm/preferences.ui:902]: Unless https://en.wikipedia.org/wiki/MIME then that is all https://en.wikipedia.org/wiki/Media_type

Comment (tsujan) [reply pcmanfm/preferences.ui:902]: → https://www.freedesktop.org/wiki/Specifications/shared-mime-info-spec/

Comment (comradekingu) [reply pcmanfm/preferences.ui:902]: https://www.iana.org/assignments/media-types/media-types.xhtml am I missing something?

File: pcmanfm/preferences.ui (hunk 2 above)
Comment (tsujan) [reply pcmanfm/preferences.ui:165]: "Right click" is a compound word with a space in the middle. We don't have a compound word like "Ctrl+right click".

File: pcmanfm/preferences.ui (hunk 1 above)
Comment (tsujan) [reply pcmanfm/preferences.ui:902]: > am I missing something?

Yes, `freedesktop.org`, which we follow.

Comment (comradekingu) [reply pcmanfm/preferences.ui:902]: So it becomes valid here once freedesktop.org updates their wording?

File: pcmanfm/preferences.ui (hunk 2 above)
Comment (comradekingu) [reply pcmanfm/preferences.ui:165]: Could you rephrase that? How I understood it doesn't appear valid to me.

File: pcmanfm/preferences.ui (hunk 1 above)
Comment (tsujan) [reply pcmanfm/preferences.ui:902]: There's nothing to update. "image/jpeg is an example of a MIME type where image is the media type, and jpeg is the subtype identifier." (→ https://help.gnome.org/admin/system-admin-guide/stable/mime-types.html.en)

File: pcmanfm/preferences.ui (hunk 2 above)
Comment (tsujan) [reply pcmanfm/preferences.ui:165]: > Could you rephrase that?

"2 red apples" vs. "2red apples"

File: pcmanfm/preferences.ui (hunk 1 above)
Comment (comradekingu) [reply pcmanfm/preferences.ui:902]: Yes, but from my understanding the word to reference the concept changed from "MIME type" to "media type" a while ago, regardless of underlying components.
https://www.iana.org/assignments/media-types/media-types.xhtml was updated 8 days ago.

There is a "top-level media type" and "media subtype" in https://www.rfc-editor.org/rfc/rfc2046.html from 1996

File: pcmanfm/preferences.ui (hunk 2 above)
Comment (comradekingu) [reply pcmanfm/preferences.ui:165]: So you are saying it should be "Ctrl+ right click"?
